AcmeClient.php

```diff
@@ -253,13 +266,24 @@ public function finalizeOrder(CertificateOrder $order, CertificateRequest $csr,
             throw new CertificateRequestFailedException('The order has not been validated');
         }
 
-        $response = $client->request('POST', $response['certificate'], $client->signKidPayload($response['certificate'], $this->getResourceAccount(), null), false);
-        $certificatesChain = null;
-        foreach (array_reverse(explode("\n\n", $response)) as $pem) {
-            $certificatesChain = new Certificate($pem, $certificatesChain);
+        if ($order->shouldReturnAlternateCertificateIfAvailable()) {
```

Comment:
adding an option to the `certificateOrder` model that will request  an alternate certificate if available... this workflow will then try to retrieve the alternate certificate, but if not available it will fall back to the default cert